frontend/src/components/layout/radiogroup/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { useState } from "react";
+import { RadioGroup } from "@headlessui/react";
+import cx from "classnames";
+
+import { CheckIcon } from "src/components/icons/CheckIcon";
+
+const difficulties = [
+  {
+    name: "easy",
+    description:
```

Comment:
descriptions currently are placeholders until we can think of something better to replace it with -- leetcode currently doesnt "describe" what each difficulty level means so we can prolly make our own definition 

* Easy: Everyone starts here.
* Medium: Once you're comfortably answering questions here, you're well on your way to securing a good internship.
* Hard: The world is your oyster -- FAANG, HFT, IB, you name it.